lib/src/nanotdf/models/ResourceLocator.ts

```diff
@@ -116,7 +175,16 @@ export default class ResourceLocator {
     } else {
       throw new Error(`Cannot create URL from protocol, "${ProtocolEnum[this.protocol]}"`);
     }
-
     return `${protocol}://${this.body}`;
   }
+
+  /**
+   * Get Identifier
+   *
+   * Returns the identifier of the ResourceLocator or an empty string if no identifier is present.
+   * @returns { string } Identifier of the resource locator.
+   */
+  getIdentifier(): string {
```

Comment:
maybe 

```suggestion
  get kid(): string {
```

Reply:
oh, since ResourceLocator is used in three places - like Remote Policy, getIdentiifer() has to be more generic